src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -19,11 +19,6 @@ TODO: review existing tests and merge with this plan:
 >     - x= every texture format
 > - 3d, {width, height, depthOrArrayLayers} > whatever the max is
 >     - x= every texture format
-> - usage flags
->     - {0, ... each single usage flag}
->     - x= every texture format
-> - every possible pair of usage flags
->     - with one common texture format
```

Comment:
The tests is not exactly the same with this TODO. I added more tests for different dimension types (1D, 2D, 3D). And I tested pair of usage flags (combined usages) for every single formats, not only a few typical formats as the TODO said. I am not sure whether it is over-tested, though. But I remember Kai said that 100% full coverage is more important for CTS, and sometimes it might be a little redundant/duplicated.

Reply:
I think the current generalization looks good. Do we need to add a sampleCount as well? I think the sampleCount != 1 disallows storage.

Reply:
It could be a different test too if that makes this one too big.

Reply:
Yeah. Here is the separate test for sampleCount: https://github.com/gpuweb/cts/pull/478. Those tests can cover sampleCount != 1 and usage includes STORAGE.